_posts/2019-07-04-exploring-nservicebus-integration-testing-options.md

```diff
@@ -5,7 +5,7 @@ layout: post
 header_image: /img/posts/exploring-nservicebus-integration-testing-options/header.jpg
 title: "Exploring NServiceBus Integration Testing options"
 author: Mauro Servienti
-synopsis: "Sometimes there is the need to test the full business scenario. Like when rehearsing for a show the entire choreography is tested. When using messaging, business scenarios testing is not easy and NServiceBus unit testing capabilities don't help much."
```

Comment:
Quick comment on changing "don't help much" to "aren't good enough". Without context, "don't help much" is often interpreted sarcastically as "The unit testing capabilities make this harder than it needs to be" which I don't think is what you intended.